flloat/base.py

```diff
@@ -47,11 +68,54 @@ def __str__(self):
         """Get the string representation."""
         return str(self.s)
 
-    def find_labels(self) -> Set[Symbol]:
+    def find_labels(self) -> Set[AtomSymbol]:
         """Return the set of symbols."""
         return {self.s}
 
 
+class QuotedFormula(Hashable):
+    """This object is a constant representation of a formula.
+
+    Quoted formulas can be used as hashable objects and for atomic symbols.
+    """
+
+    _mutable = ["_hash"]
+
+    def __init__(self, f: Formula):
+        """Initialize.
+
+        :param f: formula to represent.
+        """
+        super().__init__()
+        self.__dict__["f"] = f
+
+    def _members(self):
+        return self.f
+
+    def __str__(self):
+        """Quoted formula."""
+        return '"' + str(self.f) + '"'
```

Comment:
the outcome of `str` could be hashed? But I think this change should be done in `class Formula`. BTW, again an optional change.